contrib/google.golang.org/grpc/grpc.go

```diff
@@ -24,14 +24,18 @@ import (
 )
 
 func startSpanFromContext(
-	ctx context.Context, method, operation, service string, rate float64,
+	ctx context.Context, method, operation, service string, rate float64, measured bool,
```

Comment:
Bool arguments are weird. When reading any call it makes no sense or indication of what they are doing. I would suggest instead to change this signature to:
```
func startSpanFromContext(ctx context.Context, opts ...tracer.StartSpanOption)
```
That will make it understandable at call points and simplify things a bit. What do you think?

Reply:
Maybe another good option (easier) is:
```
func startSpanFromContext(ctx context.Context, method, operation, service string, opts ...tracer.StartSpanOption)
```
Leaving out the rate and bool.

Reply:
👍